client/src/components/MutationMapper.tsx

```diff
@@ -167,6 +167,7 @@ class MutationMapper extends React.Component<IMutationMapperProps>
                         Header: HEADER_COMPONENT[ColumnId.PERCENT_BIALLELIC]
                     },
                     {
+                        id: 'expander',
```

Comment:
Technically this is not required, but id should be specified for each column. Just in case anything goes wrong.